modules/library/main/src/main/java/org/geotools/filter/Capabilities.java

```diff
@@ -2,7 +2,7 @@
  *    GeoTools - The Open Source Java GIS Toolkit
  *    http://geotools.org
  * 
- *    (C) 2002-2015, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2002-2018, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Hi, for next time, we have a resolution stating that there is no need to update the copyright header dates (we checked, has no legal value). The only requirement is to add headers on new files, or old ones that are missing them.